### DISCUSSION TITLE
Com que frequência vocês se dedicam a projetos pessoais?

### DISCUSSION THREAD

**Discussion by renedocket** [frontendbr/forum#1815] [Discussão]
Olá pessoal, tudo bem com vocês? 

Bom, geralmente se vê muito o pessoal falando de portfólio, de projetos pessoais e afins. Na opinião de vocês, isso faz toda diferença? Ou na verdade ter esses projetos seria um trunfo pra falar que você sabe algo?  

Qual a opinião de vocês sobre?

**Comment by wilsonneto-swe** [comment]
Meu primeiro emprego relevante na área foi pro causa de um projeto pessoal de ecommerce, que vive até hoje e foi meu primeiro projeto fullstack... Acho bem válido, apesar de que hoje meus projetos pessoais se resumem a POCs e testes feitos em cursos. Separo um tempo legal para estudar diáriamente, projetos pessoais brinco de fds quando dá, mas nada relevante como no começo da carreira hehe

**Comment by mstuttgart** [comment]
Olá, @renedocket. Tudo tranquilo?

Em minha experiência pessoal, sim, faz toda a diferença!

Assim como o @wilsonneto-dev comentou, projetos pessoais podem abrir muitas portas pra você. No meu caso em particular, além de servir como uma vitrine das coisas que aprendi, os projetos também me trouxeram mais autoconfiança e uma aprendizagem enorme e em menor tempo do que seu apenas ficasse com o que eu aprendia diariamente no trabalho. Hoje em dia eu me dedico mais a manter alguns desses meus projetos que, felizmente, ainda são uteis pra bastante gente. :grin: 

Em entrevistas de emprego, os projetos certamente são um trunfo. Quando preciso entrevistar algum desenvolvedor para alguma vaga no trampo, uma das primeiras coisas que peço é o link do Github. É uma forma da gente ver se o candidato tem hábito de buscar conhecimentos por si próprio (demonstra proatividade) e não ficar apenas com o conhecimento adquirido no ambiente de trabalho. Em suma, com exceção do tempo - que seria usado em atividades de lazer - gastos nesse projeto, você ganha bastante coisa trabalhando em projetos pessoais.

Uma dica de leitura que tive recentemente e que vai de encontro a sua pergunta é o livro [O poder do tempo livre](https://www.amazon.com.br/gp/product/B075RCTBYC/ref=ppx_yo_dt_b_d_asin_title_o03?ie=UTF8&psc=1), onde o autor comenta bastante sobre o valor de criar projetos pessoais! :+1:

**Comment by ninetails** [comment]
Acho que seria legal é fazer os dois, porém não confundir (juntando dois assuntos): estudos pessoais e projetos pessoais. E ir balanceando de acordo com sua carreira.

Hoje eu não tenho tantos projetos pessoais, mas já participei de projetos como uma Radio Online e bem no começo, fórums de discussão (ambos entrando em questões mais técnicas como ajudar no site, codar etc).

Eu pessoalmente não tenho uma alma empreendedora, não sou muito de ir atrás de algo que me traga mais capital. Mas recomendo muito se a pessoa tiver interesse, tentar se aventurar em fazer algo para ganhar dinheiro como alternativa ao trabalho normal. Melhor ainda se for aprendendo também sobre empreendedorismo, gestão, fazer um bom business plan...

Por que separar projetos de estudos? Estudos a gente bem sabe que o conhecimento na nossa área é altamente volátil, muda a todo instante, logo é algo que deva ser inerente. Já projetos pessoais, não. Você faz um projeto por interesse próprio ou pra aproveitar buracos de mercado aproveitando oportunidades. E como bem disseram, isso traz reconhecimento de terceiros (na hora da entrevista, por exemplo), e também agrega conhecimento não necessariamente só ligado à programação. Também te força a não ficar preso a um só assunto durante o dia.

Eu atualmente não ando estudando muito novidades devido à pandemia. Mas to tentando pelo menos dar continuidade nos livros que comprei e nunca abri :laughing: E projetos pessoais, tirando iniciativas que tento ter na empresa, não tenho nenhum pois gosto de jogar a desculpinha de não ter tempo hábil pra ter um tempo pra descansar, jogar, etc. Mas o fato de ter já ajudado nesses projetos, fiz contatos e networkings e lidei com outros assuntos que não ligados a codar e que foi compensador.

**Comment by dionarodrigues** [comment]
A frequência que eu me dedico à projetos pessoais depende do momento profissional que estou passando: quando estou buscando novas oportunidades de empregos/parcerias ou aprendendo algo novo que pode agregar no meu marketing pessoal eu costumo me dedicar mais, por exemplo.

Creio que toda forma de comprovar o conhecimento é válida. Para mim que sou designer e front-end developer, ter um portfolio é essencial, pois é uma das primeiras solicitações que as empresas/clientes me fazem.

**Comment by kazzkiq** [comment]
Faz sim toda diferença.

GitHub é o Linkedin do mundo do desenvolvimento. Um profile rico em projetos ajuda as empresas interessadas em você a ter ideia logo de cara do seu conhecimento, qualidade de código, organização, etc.

**Na "pior" das hipóteses**, mesmo que você crie projetos pequenos que ninguém usa, eles vão servir como portfólio quando você estiver em um processo seletivo (como o @mstuttgart comentou).

**Na melhor das hipóteses**, se você "emplacar" um projeto, a tendência é seu perfil ganhar muita visibilidade ao ponto de chamar atenção de headhunters no Brasil e no exterior, te gerando ainda mais ofertas de emprego.

Tem startups que nós nem imaginamos fazendo "rankings" de devs com base na atividade dos perfis no GitHub, StackOverflow, etc, e oferecendo esse ranking à empresas interessadas. Então passar um tempo nessas plataformas pode ser um baita investimento pessoal.
